components/basecomponents/Exercise/components/ChoiceListComponent.tsx

```diff
@@ -0,0 +1,52 @@
+import { useLanguage } from 'utils/useLanguageHook';
+import { Choice, ExerciseStatus, resolveMethods, resultMethods, useExerciseStore } from '../exerciseStore';
+import { useState } from 'react';
+import { ChoiceComponent } from './ChoiceComponent';
+
+interface ChoiceListComponentProps {
+  choices: Choice[];
+  correctChoiceId: number;
+  status: ExerciseStatus;
+}
+
+export const ChoiceListComponent = ({ choices, correctChoiceId, status }: ChoiceListComponentProps) => {
```

Comment:
This was extracted because of code duplication but it needs to be refactored. I plan to do this, maybe today.

Reply:
Done.